src/sonic-config-engine/tests/simple-sample-graph.xml

```diff
@@ -178,12 +178,12 @@
       <PortChannelInterfaces>
         <PortChannel>
           <Name>PortChannel01</Name>
-          <AttachTo>fortyGigE0/4</AttachTo>
+          <AttachTo>Ethernet4</AttachTo>
```

Comment:
why we change here?  Any functional change by this change? same comments in the below changes in this file.

Reply:
As per the original logic, a port belonging to a port-channel will be admin up. However, the simple-sample-graph file is used for test cases between port_config.ini and platform.json (which is a very poor initial design choice). It was using the alias in port_config.ini. However, these alias were not present in platform.json (which has a defined format of alias). So the common ground between these two is to use the interface name directly.